src/components/Playbooks/Runs/Submit/PlaybookSelectResource.tsx

```diff
@@ -46,7 +49,18 @@ export default function PlaybookSelectResource({
         <div className="w-36">
           <label htmlFor={`config_id`} className="form-label mb-0 py-4">
             {resourceSelectorLabel?.label}
+            <HelpLink
+              link="reference/resource-selector#search"
+              title=""
+              className="ml-1"
+              iconID="help-resource-selector"
+            />
           </label>
+          <Tooltip
+            className="z-50"
+            anchorSelect="#help-resource-selector"
+            content="eg: name=grafana type=Kubernetes::Deployment or health=healthy,unhealthy"
```

Comment:
This should be a hint underneath the text field that is always shown
```suggestion
            content="Filter the list using queries e.g: name=grafana type=Kubernetes::Deployment or health=healthy,unhealthy"
```

And then the help icon is not a tooltop but a straight link to docs